.github/workflows/build.yml

```diff
@@ -0,0 +1,120 @@
+name: Build
+
+on:
+  push:
+  schedule:
+    - cron: '0 4 * * *'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  linux:
+    strategy:
+      matrix:
+        arch: ['ia32', 'x64']
+    runs-on: ubuntu-latest
+    container:
+      # using oldest image to ensure support for older glibc
+      # from node:12.0.0 on 2021-08-30
+      image: node@sha256:c88ef4f7ca8d52ed50366d821e104d029f43e8686120a29541ce0371f333453f
+    name: linux-${{ matrix.arch }}
+    env:
+      ARCH: ${{ matrix.arch }}
+    steps:
+      - run: apt-get update && apt-get install -y g++-multilib
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/prebuild
+
+  macos-apple:
+    runs-on: macos-11
+    name: macos-apple
+    env:
+      ARCH: arm64
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/prebuild
+
+  macos-intel:
+    strategy:
+      matrix:
+        arch: ['ia32', 'x64']
```

Comment:
Wait, we build for 32-bit mac?

Reply:
Yes, I figured I'd keep it although it's extremely unlikely to be used.